mlflow/server/js/src/utils/Utils.test.js

```diff
@@ -153,7 +153,19 @@ test("formatSource & renderSource", () => {
   };
   const wrapper2 = shallow(Utils.renderSource(databricksRun, databricksRunRevisionTags));
   expect(wrapper2.is("a")).toEqual(true);
-  expect(wrapper2.props().href).toEqual("../#notebook/13/revision/42");
+  expect(wrapper2.props().href).toEqual("http://localhost/#notebook/13/revision/42");
```

Comment:
micronit: maybe it's preferable to use these matchers? https://github.com/producthunt/chai-enzyme